x-pack/plugins/fleet/common/experimental_features.ts

```diff
@@ -32,9 +28,18 @@ export const allowedExperimentalValues = Object.freeze<Record<string, boolean>>(
   advancedPolicySettings: true,
   useSpaceAwareness: false,
   enableReusableIntegrationPolicies: false,
-});
+};
+
+/**
+ * A list of allowed values that can be used in `xpack.fleet.enableExperimental`.
+ * This object is then used to validate and parse the value entered.
+ */
+export const allowedExperimentalValues = Object.freeze<
```

Comment:
Not really related to that feature, but it allow to add some type safety to `allowedExperimentalValues`